client/src/pages/Landing/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useMediaQuery } from 'react-responsive';
+import { Row, Col } from '../../components/Grid';
+import { CTA, Search, HelpOut } from '../../components/Card';
+import * as T from '../../components/Typography';
+import * as R from '../../constants/nav-routes';
+import { breakpoints } from '../../theme';
+
+const Landing = () => {
+  const isMobile = useMediaQuery({
+    query: `(max-width: ${breakpoints.mobile})`,
+  });
+
+  return (
+    <article>
+      <T.H2 m="0" mb={isMobile ? '5' : '6'}>
+        Learning digital skills
+      </T.H2>
+      <Row mb="3">
+        <Col w={[4, 6, 4]} mb={3}>
```

Comment:
the columns by default take padding(16px form both sides ) but the gap in desktop is 24px regardless of the orders of the columns so the first column takes 16px padding from the left side and the same for the last one from the right and this make the gap 32px in the middle 
```suggestion
        <Col
           w={[4, 6, 4]}
            mb={3}
            style={{
              padding: !isMobile && '0px',
              paddingRight: !isMobile && '8px',
          }}
          >
```

Reply:
Hey this feels like something that should have been set up / be fixed then in the grid component? @RamyAlshurafa ?

Reply:
Hey @thejoefriel 
so just was with a call with @fadeomar, so you can update the `gridGutter` in the theme so it 12 instead of the 16, this will make the inner spacing 24, but the outer columns will still have 12px padding on each side, and when i build the grid component didnt find its very hard to fix this issue, as the position of the columns will not stay the same on all screens
so Fadi added padding constants for this layout as 60 px in the theme, so we can change it to 48px

Reply:
ok have reduced to 12px and changed the 60px to 48px